web_monitoring/db.py

```diff
@@ -760,3 +781,25 @@ def get_version_by_versionista_id(self, versionista_id):
         # Make result look like the result of `get_version` rather than the
         # result of `list_versions`.
         return {'data': result['data'][0]}
+
+    def validate_db_credentials(self):
+        """
+        Validate that the DB Client is authorized for the provided host
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+        none
+        """
+        try:
+            response = self.get_user_session()
+        except requests.exceptions.HTTPError as exc:
+            if exc.response.status_code == 401:
+                raise MissingCredentials(f'\n\n\n\nCheck your DB credentials - 401 Unauthorized for url: {exc.request.url}\n\n\n\n')
+            else:
+                # Client didn't receive a 401, but hit another 4XX or 5XX error
+                raise RuntimeError(f'\n\n\n\nOops: an unexpected error occurred while checking DB credentials...\n\n{exc}\n\n\n\n')
```

Comment:
Similarly, I think it’s OK to just let other exceptions raise here without special handling. Providing nicer messaging should go in `validate_db_credentials()` in `cli.py`.

I think the ideal implementation for this whole method might just be:

```py
def validate_credentials(self):
    self.get_user_session()
```

And all the error handling should move to different places.